model.py

```diff
@@ -4836,24 +4836,55 @@ def scaled_weight(self):
     ])
 
     _quality_as_indicator_of_target_age = {
-        # These measure age appropriateness.
-        (DataSource.METADATA_WRANGLER, Subject.AGE_RANGE) : 100,
-        Subject.AXIS_360_AUDIENCE : 30,
-        (DataSource.OVERDRIVE, Subject.GRADE_LEVEL) : 20,
-        (DataSource.OVERDRIVE, Subject.INTEREST_LEVEL) : 20,
-        Subject.OVERDRIVE : 15,
-        (DataSource.AMAZON, Subject.AGE_RANGE) : 10,
-        (DataSource.AMAZON, Subject.GRADE_LEVEL) : 9,
-
-        # These measure reading level, except for TAG, which measures
-        # who-knows-what.
-        (DataSource.OVERDRIVE, Subject.GRADE_LEVEL) : 5,
-        Subject.TAG : 2,
-        Subject.LEXILE_SCORE : 1,
-        Subject.ATOS_SCORE: 1,
-
-        Subject.AGE_RANGE : 10,
-        Subject.GRADE_LEVEL : 10,
+        
```

Comment:
Previously these numbers were just relative rankings and had no special significance. Now they are coefficients ranging from 0 to 1 and expressing how confident we are in the reliability of a target age indicator.